### PR TITLE
Allow all specs and code lints to be run locally

### PR DESCRIPTION
**Pull request by eoinkelly**
This change allows a developer to run the same lints and specs that Travis, Codeclimate and Deppbot will run. This allows us to find problems before pushing (so hopefully less noise for reviewers).

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ module.exports = {
   extends: 'google',
   env: {
     node: true,
+    browser: true,
     es6: false
   },
+  rules: {
+    "no-var": "off",
+    "require-jsdoc": "off"
```
Comment (eoinkelly) [inline .eslintrc.js:10]: These two rules were moved from .codeclimate.yml

**Review by Br3nda — APPROVED** [review]
Approved. Thank you for your mahi.

In other projects I've used the overcommit gem. It becomes a standard way to run linters, and remind devs to rerun them.

**Comment by Rabid-Dan** [comment]
👏 👏 👏

File: .eslintrc.js (hunk 1 above)
Comment (louise-r-blue) [reply .eslintrc.js:10]: @eoinkelly This is such a sensible change. Will the README be updated to reflect the changes. Something like `scripts described here` to run locally should be executed locally before pushing up a branch.

Comment (eoinkelly) [reply .eslintrc.js:10]: I did add to the README about this https://github.com/ODNZSL/nzsl-online#running-the-tests - is that what you mean?

Comment (louise-r-blue) [reply .eslintrc.js:10]: Oh ok ! I'm a few steps behind.

**Pull request merged**